package.json

```diff
@@ -309,5 +317,9 @@
     "vite-tsconfig-paths": "^5.0.1",
     "vitest": "^2.1.3",
     "yaml": "^2.6.0"
+  },
+  "optionalDependencies": {
+    "@google-cloud/vertexai": "^1.9.0",
+    "google-auth-library": "^9.15.0"
   }
```

Comment:
This should not be needed.

Reply:
I can not figure out how to set only in the `peerDependencies` and  `peerDependenciesMeta`.  I added these modules using 
```
yarn add xxxx -P -O
```
What is the right way to add these?  
Thanks!

Reply:
Add them manually and then just run `yarn`.